modules/va_facilities/README.yml

```diff
@@ -64,22 +66,22 @@ info:
 tags:
   - name: facilities
     description: VA Facilities API
-servers:
-  - url: https://dev-api.va.gov/services/va_facilities/{version}
-    description: VA.gov API development environment
-    variables:
-      version:
-        default: v0
-  - url: https://staging-api.va.gov/services/va_facilities/{version}
-    description: VA.gov API staging environment
-    variables:
-      version:
-        default: v0
-  - url: https://api.va.gov/services/va_facilities/{version}
-    description: VA.gov API production environment
-    variables:
-      version:
-        default: v0
+# servers:
```

Comment:
Is there a reason not to simply delete this section? Will we be using it later?

Reply:
Solid point. I can remove it unless there's a reason to keep it around. It would only be useful again should we move to a new renderer other than swagger-ui. But I looked through the others team's docs and they've just removed.

Reply:
Ok, good to know. Would you like me to remove it here and in the v1 readme?

Reply:
I took care of it. Thanks!